### ISSUE TITLE
not compatible with node v0.11

### ISSUE THREAD

**Issue by ben-page** [nathanjsweet/nodehun#15]
nodehun isn't compatible with node v0.11 or the soon to be released v0.12. The easiest way to accomplish this is with this: [NAN](https://github.com/rvagg/nan/)

**Comment by ben-page** [comment]
Why not be ready when it releases? Many people are already using v0.11 in production. It's pretty much your only choice if you use node-webkit. 

Plus, v0.11.15 considered the [release candidate](https://twitter.com/nodejs/status/545349270241435648) for v0.12. The [API changes](https://github.com/joyent/node/wiki/API-changes-between-v0.10-and-v0.12) have been available for months. Moreover, if you choose to use [NAN](https://github.com/rvagg/nan/) you wont have to worry about future API changes.

**Comment by nathanjsweet** [comment]
The only thing I'm using mentioned in the changes is Buffer, but I never instantiate it. From what I can see in the code I should be okay.

**Comment by reznikartem** [comment]
Hello! Can i remind you about this problem? :)
Thank you!

**Comment by reznikartem** [comment]
Great news!

**Comment by nathanjsweet** [comment]
tldr; This works now.

Okay, this is up to date (pushing to npm in a few minutes). PLEASE make sure to pay attention to my notes on versioning. Version 1.x.x is for node prior to v0.12.0 and version 2.x.x works on v0.12.0 and greater. I also did a little research on v8 and noticed that the node docs use methods for obtaining isolates that the v8 team discourages using, because they are going to become deprecated soon, so hopefully this little module will stay future-proof for longer.

I decided against using Nan as the documentation was not easy to follow, and I don't necessarily trust that the next v8 changes will be easy to wrap.

**Comment by ben-page** [comment]
Great work. Thank you!
